src/OmniSharp/Models/v1/DnxProject.cs

```diff
@@ -12,6 +13,7 @@ public class DnxProject
         public IList<string> Configurations { get; set; }
         public IList<string> ProjectSearchPaths { get; set; }
         public IList<string> Frameworks { get; set; }
+        public IList<DnxFramework> DnxFrameworks { get; set; }
```

Comment:
It's used by @vscode which is why I cc'd @jrieken to see if he'd be ok with changing this.